Pollo/ViewControllers/PollBuilderViewController.swift

```diff
@@ -53,10 +53,6 @@ class PollBuilderViewController: UIViewController {
     var questionType: QuestionType!
     var shouldIgnoreNextKeyboardHiding: Bool = false
     weak var delegate: PollBuilderViewControllerDelegate?
-
-    override var preferredStatusBarStyle: UIStatusBarStyle {
```

Comment:
Was this related to what we talked about with iOS 13.4? Will it still work for lower versions?

Reply:
it doesn't work for iOS 13+ so i replaced it by setting `navigationController?.navigationBar.barStyle = .black` which forces white text in the status bar. i think it has to do with DarkMode